From a12a3b981edde5eee91104c892ec4be485c5c9e7 Mon Sep 17 00:00:00 2001
From: Yuchou Gan <yuchou.gan@nxp.com>
Date: Fri, 9 Mar 2018 22:37:15 +0800
Subject: [PATCH 3458/5242] MGS-3724 increase the core clock rate for qxp B0
 board.

commit  d3830884dffa4eba115cafe951a291df97ea62f2 from
https://source.codeaurora.org/external/imx/linux-imx.git

The qxp B0 board gpu core clock rate is 700MHz, increase to it.

Signed-off-by: Yuchou Gan<yuchou.gan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
index 0ac2271..f7dde18 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
@@ -1974,7 +1974,7 @@
 		clocks = <&clk IMX8QXP_GPU0_CORE_CLK>, <&clk IMX8QXP_GPU0_SHADER_CLK>;
 		clock-names = "core", "shader";
 		assigned-clocks = <&clk IMX8QXP_GPU0_CORE_CLK>, <&clk IMX8QXP_GPU0_SHADER_CLK>;
-		assigned-clock-rates = <600000000>, <850000000>;
+		assigned-clock-rates = <700000000>, <850000000>;
 		power-domains = <&pd_gpu0>;
 		status = "disabled";
 	};
-- 
1.7.9.5

